setup.py

```diff
@@ -13,18 +20,43 @@ def get_description():
         return info + '\n\n' + changelog.read()
 
 
-with open(loc + '/requirements.txt') as f:
+with open(loc + '/requirements-micro.txt') as f:
     required = f.read().splitlines()
+if not micro:
+    with open(loc + '/requirements.txt') as f:
+        for line in f:
```

Comment:
this can be a one-liner:
`required.extend([l for l in f.read().splitlines() if l != '-r requirements-micro.txt'])`

Reply:
Done